<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UDP</class>
 <widget class="QDialog" name="UDP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>109</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UDP参数配置</string>
  </property>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>25</y>
     <width>48</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>本机IP：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="UDPSenderPortlineEdit">
   <property name="geometry">
    <rect>
     <x>221</x>
     <y>20</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>2501</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_27">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>端口：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>端口：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="UDPFarPortlineEdit">
   <property name="geometry">
    <rect>
     <x>221</x>
     <y>50</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>2500</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>55</y>
     <width>48</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>目标IP：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="UDPServerIPcomboBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="UDPFarIPlineEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>169.254.25.129</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OKpushButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确 定</string>
   </property>
  </widget>
  <widget class="QLabel" name="UDPServerStatelabel">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>80</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Ready</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
